app/views/configuration/_ui_2.html.haml

```diff
@@ -50,7 +50,8 @@
                       :containerimage         => "container_image",
                       :containerimageregistry => "container_image_registry",
                       :container              => "container",
-                      :containertopology      => "container_topology"}
+                      :containertopology      => "container_topology",
+                      :container_dashboard    => "container_dashboard"}
```

Comment:
not sure if by mistake or on purpose, but all names here are without underscore, while container_dashboard has an underscore